core/_neat.scss

```diff
@@ -0,0 +1,11 @@
+// Neat 2.0.0.alpha.0
+// http://neat.bourbon.io
+// Copyright 2015 thoughtbot, inc.
+// MIT License
+
+@charset "UTF-8";
```

Comment:
Also, in Bourbon we put these in each individual partial. I can’t speak to if one way is right or wrong, but it solved an issue we had with using special characters in our SassDoc comments, like em dashes and things.